proto_quic_tools/update.py

```diff
@@ -113,12 +113,18 @@ def copy_modified_files():
 
 # Since proto-quic users do not use gclient sync, runs necessary parts of it.
 def sync():
-  command = "/" + proto_quic_root + "/third_party/binutils/download.py"
+  command = "/" + proto_quic_root + "proto_quic_tools/sync.sh"
```

Comment:
I notice we're prepending "/" to PROTO_QUIC_ROOT. Isn't it expected that PROTO_QUIC_ROOT is already an absolute path?